docs/source/tutorials/staging_data.ipynb

```diff
@@ -1,20 +1,340 @@
 {
```

Comment:
<p>Please merge the cells into a single code cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I would propose to have such splits in code, since it is a tutorial notebook, it is good to have code cells with modules import and initialization of instances separately from other parts of the code. In this examples user will run initialization of Session, PilotManager and TaskManager once, but will "play" with pilots and tasks by rerunning that cells. We create a playground for the test examples.</p>